api/swagger.yml

```diff
@@ -1339,152 +1512,138 @@ paths:
         204:
           description: repository deleted successfully
         401:
-          $ref: "#/responses/Unauthorized"
+          $ref: "#/components/responses/Unauthorized"
         404:
-          description: repository not found
-          schema:
-            $ref: "#/definitions/error"
+          $ref: "#/components/responses/NotFound"
         default:
-          description: generic error response
-          schema:
-            $ref: "#/definitions/error"
+          $ref: "#/components/responses/ServerError"
 
   /repositories/{repository}/refs/dump:
     parameters:
       - in: path
         name: repository
         required: true
-        type: string
+        schema:
+          type: string
     put:
       tags:
         - refs
-      operationId: dump
+      operationId: dumpRefs
```

Comment:
Previously we relied on the first tag to give a prefix.  If we now need to repeat it, consider putting the tag at the beginning for better scoping.

Reply:
Used Oz's conversion - it looks like we wanted to be more like the operation itself in the code than a prefix.
Using the tag as prefix in the code can be modified by the template - but for now if you check the generated code it much cleaner.